### PR TITLE
DIT-375 Reject case

### PR DESCRIPTION
**Pull request by pllaurado**

### CODE REVIEW

**Comment by gokyo** [comment]
@pllaurado 
Can you add a scenario of your action in `AuditServiceTest` ?

**Comment by gokyo** [comment]
@pllaurado 
Can you create a file `CasesService_RejectCaseSpec.scala` and add similar tests as done for the other actions?

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/reject_suspend_section.scala.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.models.{Case, CaseStatus}
+@(c: Case)
+
+@if(c.status == CaseStatus.OPEN) {
+    <div class="mt-1">
```
Comment (ed0906) [inline app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/reject_suspend_section.scala.html]: Imagine its worth sticking an ID on this so the logic can be tested?

**Comment by ed0906** [comment]
I think there's some `ViewSpec` missing

**Comment by ed0906** [comment]
Looks fine to me, I'd pull in master first though given we're all working in the same area

**Pull request merged**